server/models/articles.model.js

```diff
@@ -1,13 +1,44 @@
 const db = require("../../db/connection");
 
+exports.fetchArticles = () => {
+  return db
+    .query(
+      `SELECT articles.article_id, articles.title, articles.topic, articles.author, articles.created_at, articles.votes, articles.article_img_url, 
+      COUNT(comments.article_id) AS comment_count
+      FROM articles 
+      LEFT OUTER JOIN comments ON articles.article_id = comments.article_id
+      GROUP BY articles.article_id
+      ORDER BY created_at DESC`
+    )
+    .then(({ rows }) => {
+      return rows;
+    });
+};
+
 exports.fetchArticle = (article_id) => {
   return db
     .query(`SELECT * FROM articles WHERE article_id = $1;`, [article_id])
     .then((result) => {
       if (result.rows.length === 0) {
-        return Promise.reject({ status: 404, msg: "team does not exist" });
+        return Promise.reject({ status: 404, msg: "article does not exist" });
       } else {
         return result.rows[0];
       }
     });
 };
+
+exports.fetchCommentsByArticleId = (article_id) => {
+  return db
+    .query(
+      `SELECT * FROM comments WHERE article_id = $1
+    ORDER BY created_at DESC;`,
+      [article_id]
+    )
+    .then((result) => {
+      if (result.rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "article does not exist" });
```

Comment:
You don't want this check/error here because you want to return a 200 if there are no comments. This is where you might use a "checkIfExists" model to separate throwing an error if the article doesn't exist, but allowing a 200 with an empty array for if the article _does_ exist but doesn't have any comments.